eng/tools/GenerateFiles/Directory.Build.targets.in

```diff
@@ -64,7 +64,7 @@
 
   <ItemGroup>
     <!-- Use the same NETCore shared framework as repo built against except when building product code in servicing. -->
-    <KnownFrameworkReference Update="Microsoft.NETCore.App">
+    <KnownFrameworkReference Update="Microsoft.NETCore.App" Condition=" '$(PackAsToolShimRuntimeIdentifiers)' == '' ">
```

Comment:
Is the condition on the framework reference necessary?

Reply:
Oops - absolutely - this should only apply in VMR builds. Thanks - will fix.

Reply:
Yeah, we do need it. Otherwise we get this error:
```
error NU1102: Unable to find package Microsoft.NETCore.App.Runtime.win-x86
```
Missing apphost pack was causing the same error, for the other package:
```
error NU1102: Unable to find package Microsoft.NETCore.App.Host.win-x86
```

It was also suggested in your comment in the original issue: https://github.com/dotnet/source-build/issues/4944#issuecomment-2730576574